_docs/configuration_files/application_json.rst

```diff
@@ -152,6 +145,13 @@ Minimum number of instances your auto-scaling group should have at all times. Th
     | *Type*: int
     | *Default*: ``1``
 
+``ssh_keypair``
+*******************
+
+SSH key that your EC2 instances will use. Must already be created in AWS.
```

Comment:
I will add this just to be clear, but state that it never worked. Migrate to this.